<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>omg its a 3D viewwerw</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="display" name="openGLWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_openFile">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
    color: rgb(16, 19, 26);
    background-color: rgb(255, 118, 21);
    margin: 0px 12px;
    border-radius: 8px; /* Добавляет скругленные углы */
    padding: 5px 15px; /* Добавляет отступы внутри кнопки */
}

QPushButton:hover {
    background-color: rgb(255, 135, 45); /* Изменяет цвет при наведении */
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
    stop: 0 #FF6723, stop: 1 #FF9635); /* Более выраженный градиент при нажатии */
    padding-top: 6px; /* Смещение вниз при нажатии для эффекта нажатия */
    padding-left: 14px; /* Легкое смещение влево для эффекта нажатия */
}
</string>
          </property>
          <property name="text">
           <string>OPEN FILE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_reload">
          <property name="text">
           <string>Reload</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="label_filename_2">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Filename:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_filename">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(25, 94, 255);
font-size: 16px</string>
            </property>
            <property name="text">
             <string>No file</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Vertices:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_vert">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Edges:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_face">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_screen">
          <property name="text">
           <string>Screen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_gif">
          <property name="text">
           <string>GIF</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_timer">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionView_preferences"/>
   <addaction name="actionModel_interactions"/>
  </widget>
  <action name="actionView_preferences">
   <property name="text">
    <string>View_preferences</string>
   </property>
  </action>
  <action name="actionModel_interactions">
   <property name="text">
    <string>Model_interactions</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>display</class>
   <extends>QOpenGLWidget</extends>
   <header>display.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
